src/main/java/org/jvnet/hudson/test/JenkinsRule.java

```diff
@@ -2246,5 +2361,9 @@ public Description getTestDescription() {
         return testDescription;
     }
 
+    private NameValuePair getCrumbHeaderNVP() {
+        return new NameValuePair(jenkins.getCrumbIssuer().getDescriptor().getCrumbRequestField(),
+                        jenkins.getCrumbIssuer().getCrumb( null ));
```

Comment:
I know it's copied from below, but you are inheriting the sin :smile_cat: (spaces around `null`).